PythonTool/PlotContour.py

```diff
@@ -0,0 +1,31 @@
+from visit import *
+
+
+def PlotContour(File, ObjectSequence):
+    # Contour plot attributes.
+
+    # Change orientation of legend
+    LegendAttributes = GetAnnotationObject(ObjectSequence["Contour"])
+    LegendAttributes.orientation = "VerticalLeft"
+    LegendAttributes.fontHeight = 0.03
+
+    Attribute = ContourAttributes()
+
+    Attribute.legendFlag = 1
+
+    if File is not None:
```

Comment:
The reason they were similar was because I was playing with the options available in each plot and saw if I could change them individually. Basically, invoke one without the other. There may be some added attributes for mesh and contour that may make them dissimilar in the future.